pkgs/by-name/gn/gnunet-messenger-cli/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+  meson,
+  ninja,
+  pkg-config,
+  gnunet,
+  libsodium,
+  libgcrypt,
+  libgnunetchat,
+  ncurses,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gnunet-messenger-cli";
+  version = "0.3.1";
+
+  src = fetchgit {
+    url = "https://git.gnunet.org/messenger-cli.git";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-8Iby3IZXEZJ1dqVV62xDzXx/qq7JKhVtn6ZLb697ZSw=";
+  };
+
+  env.INSTALL_DIR = (placeholder "out") + "/";
+
```

Comment:
sorry to nitpick, but consider adding `strictDeps = true;` its generally considered to be a good practice nowadays + helps ensure cross compilation works.